openeo/extra/job_management.py

```diff
@@ -691,6 +699,25 @@ def __init__(self):
         super().__init__()
         self._df = None
 
+    def initialize_from_df(self, df: pd.DataFrame):
```

Comment:
also have a mode/option so that users can also call it when the db already exists, but instead of exception:
- skip initialization
- or merge data,
- or ...

Reply:
done